cloudflare-mixin/signals/geomap.libsonnet

```diff
@@ -0,0 +1,39 @@
+function(this) {
+  filteringSelector: this.filteringSelector,
+  groupLabels: this.groupLabels,
+  instanceLabels: this.instanceLabels,
+  aggLevel: 'none',
+  aggFunction: 'avg',
+  alertsInterval: '2m',
+  discoveryMetric: {
+    prometheus: 'cloudflare_zone_requests_country',
+  },
+  signals: {
+    geoMapByCountry: {
+      name: '$geo_metric Distribution',
```

Comment:
Maybe I'm still mentally on vacation, but I don't recall seeing this used in the name before, just wanted to call out that it's a cool usage!